Directory.Packages.props

```diff
@@ -6,6 +6,7 @@
 -->
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
+    <PackageVersion Include="Microsoft.ProjectReunion.InteractiveExperiences.TransportPackage" Version="1.8.0-CI-26107.1720.241003-1631.3" />
```

Comment:
Did this get automatically added when you added a reference to this dependency for the IXP packagereference?

Reply:
I made change to version.details.xml and synced dependencies.